UM/Trust.py

```diff
@@ -36,6 +36,16 @@ class TrustBasics:
     __signature_filename_extension = ".signature"
     __root_signature_entry = "root_signature"
 
+    @staticmethod
```

Comment:
This entire method confuses me. Probably the docstring is outdated. 

Why raise a SystemExit with an obscure hex arg instead of using sys.exit()? Will it be caught to allow some graceful cleanup / shutdown logic?

Reply:
Should it even break as hard as it's doing right now? If a single file is not trusted, we shouldn't shut down the entire application (Because that would mean that if one file gets corrupted, the entire application doesn't start!). It would take a single plugin to be placed in the local folder to prevent someome from using cura.

This is also why our unit tests are failing; They simply expect that the function returns that the file is not trusted and that the program just continues from that point onwards.

Reply:
The idea was to use another handler when Trust is used in another way (like for material _verification_ instead of _security_). Since we're not going to crash anymore at all the point is a bit moot.

Reply:
Also for the unit-tests, so the tests wouldn't crash while running.